.jcheck/conf

```diff
@@ -38,7 +38,7 @@ project=jmc
 jbs=jmc
 
 [checks]
-error=author,reviewers,merge,message,issues,executable
+error=author,reviewers,merge,message,issues
```

Comment:
If you want to retain the executable check, you could prefix the commands with `sh`..

Reply:
Ok. I find it convenient to have these scripts executable locally. I have no real strong opinions on this though. If someone cares strongly either way, please let me know.